src/redux/reducers/bananaReducer.js

```diff
@@ -0,0 +1,15 @@
+function bananaReducer(state = {missed: 0, caught: 0}, action) {
+    switch (action.type) {
+        case 'CAUGHT_BANANA': {
+            state.caught ++;
+            return {...state, caught: state.caught ++};
+        }
+        case 'MISSED_BANANA': {
+            state.missed ++;
+            return {...state, missed: state.missed ++};
+        }
+        default:
+            return state;
+    }
+}
+export default bananaReducer;
```

Comment:
Since you are using `redux-toolkit` you should look into the `createSlice`, `createAction`, and `createReducer` hooks.  They make `redux` a lot cleaner, which a lot less switch statements like this.  Also look into thunk middleware for async stuff.  It replaces Sagas and is way way easier.